### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by Kevin2**

### CODE REVIEW

**Comment by Kevin2** [comment]
rename repo to be able to publish it on pypi

File: pyvisgen/fits/writer.py [hunk 1]
```diff
@@ -370,7 +373,7 @@ def create_antenna_hdu(layout_txt, conf, layout="EHT"):
     return hdu_ant
 
 
-def create_hdu_list(data, conf, path="../vipy/layouts/eht.txt"):
+def create_hdu_list(data, conf, path="../pyvisgen/pyvisgen/layouts/vlba.txt"):
```
Comment (FeGeyer) [inline pyvisgen/fits/writer.py:376]: That should be a `pyvisgen` too much

File: pyvisgen/simulation/utils.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def read_config(conf):
     sim_conf["src_coord"] = SkyCoord(
         ra=config["sampling_options"]["fov_center_ra"],
         dec=config["sampling_options"]["fov_center_dec"],
-        unit=(u.hourangle, u.deg),
+        unit=(u.deg, u.deg),
```
Comment (FeGeyer) [inline pyvisgen/simulation/utils.py:27]: Why is that changed? Was it a bug?

**Review by FeGeyer — CHANGES_REQUESTED** [review]

**Comment by FeGeyer** [comment]
Maybe rework the `README` before adding this to `pypi`.
